openflow13/action.go

```
@@ -262,6 +325,38 @@ type ActionMplsTtl struct {
 	pad     []byte // 3bytes
 }
 
+type ActionDecNwTtl struct {
+	ActionHeader
+	pad []byte // 4bytes
+}
+
+func NewActionDecNwTtl() *ActionDecNwTtl {
+	act := new(ActionDecNwTtl)
+	act.Type = ActionType_DecNwTtl
+	act.Length = 8
+	act.pad = make([]byte, 4)
+	return act
+}
+
+func (a *ActionDecNwTtl) Len() (n uint16) {
+	return a.ActionHeader.Len() + 4
+}
+
+func (a *ActionDecNwTtl) MarshalBinary() (data []byte, err error) {
+	data, err = a.ActionHeader.MarshalBinary()
+
+	// Padding
+	bytes := make([]byte, 4)
+
+	data = append(data, bytes...)
+	return
+}
+
+func (a *ActionDecNwTtl) UnmarshalBinary(data []byte) error {
+	a.ActionHeader.UnmarshalBinary(data[:4])
+	return nil
```

Please return the error from the called function.